headshots.py

```diff
@@ -29,13 +36,11 @@
         # ESC pressed
         print("Escape hit, closing...")
         break
-    elif k%256 == 32:
+    if k%256 == 32:
         # SPACE pressed
-        img_name = "dataset/"+ name +"/image_{}.jpg".format(img_counter)
+        img_name = DATASET_FOLDER+"/"+ NAME +"/image_{}.jpg".format(img_counter)
```

Comment:
pylint is insisting on treating this variable as a constant, so i recommend adding this comment to the end of the line: `# pylint: disable=invalid-name`